backend/tests/test_webdav_api.py

```
@@ -45,3 +65,157 @@ def test_get_project_folders(auth_client):
+                    )
+                    """
+                ),
+                {
+                    "account_id": 8871,
+                    "user_id": "alice",
+                    "server_url": "https://real-webdav.naruon.net",
+                    "username": "alice",
+                    "credentials_encrypted": "test-only-placeholder",
+                },
+            )
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)